packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -1665,25 +1665,6 @@ const provisioningStateSpecified = (pathItem, _opts, ctx) => {
     return errors;
 };
 
-const scopeParameter = "{scope}";
-const noDuplicatePathsForScopeParameter = (path, _opts, ctx) => {
-    var _a;
-    const swagger = (_a = ctx === null || ctx === void 0 ? void 0 : ctx.documentInventory) === null || _a === void 0 ? void 0 : _a.resolved;
-    if (path === null || typeof path !== "string" || path.length === 0 || swagger === null) {
-        return [];
-    }
-    const pathRegEx = new RegExp(path.replace(scopeParameter, ".*"));
-    const otherPaths = Object.keys(swagger.paths).filter((p) => p !== path);
-    const matches = otherPaths.filter((p) => pathRegEx.test(p));
-    const errors = matches.map((match) => {
-        return {
-            message: `Path "${match}" with explicitly defined scope is a duplicate of path "${path}" that has the scope parameter.".`,
```

Comment:
>            message: `Path "${match}" with explicitly defined scope is a duplicate of path  [](http://example.com/codeflow?start=0&length=91)

Do we checkin this file? I thought this is autogenerated & never checked it in.

Reply:
It is generated when you build the project. I'm not sure if it's necessary or not, but Roopesh [checked it in](https://github.com/Azure/azure-openapi-validator/commit/856b31b6c6d560e653b6e5fecd9e966c46563cfd#diff-2c4dbe936f4558f78f5611dc25eaba744271ce8a45191f1d92143dbd653be0cf) for at least one of his linter rules. Full history is [here](https://github.com/Azure/azure-openapi-validator/commits/main/packages/rulesets/generated/spectral/az-arm.js).

Reply:
I think it probably does not matter in the end because the project will be rebuilt in the pipeline when it runs.